backend/models/user.js

```diff
@@ -109,14 +171,18 @@ User.authenticate =  async function (logger, username, password) {
 			username = user.user;
 		}
 
-		authDB = await DB.getAuthDB();
-		await authDB.authenticate(username, password);
-		authDB.close();
-
 		if (!user)  {
 			user = await User.findByUserName(username);
 		}
 
+		if (hasReachedFailedLoginLimit(user)) {
```

Comment:
I think this is better described as `isAccountLocked`

Coz you can have reached the failed login count, but the lockout time has expired.